src/Resolver/ValueResolver/ValueResolver.php

```diff
@@ -134,6 +150,22 @@ public function resolveStrings(Expr $expr, Scope $scope): ?array
         if ($values === null) {
             return null;
         }
+
         return array_filter($values, 'is_string');
     }
+
+    /**
+     * @return (string|int)[]|null
```

Comment:
`array<int|string>` is more readable than `(int|string)[]` I think

Reply:
I can change it